res/css/views/settings/_SetIntegrationManager.scss

```diff
@@ -15,22 +15,26 @@ limitations under the License.
 */
 
 .mx_SetIntegrationManager {
-    margin-top: 10px;
-    margin-bottom: 10px;
-}
+    .mx_SettingsFlag {
+        align-items: center;
+        margin-top: var(--SettingsTab_heading_nth_child-margin-top);
 
-.mx_SetIntegrationManager > .mx_SettingsTab_heading {
-    margin-bottom: 10px;
-}
+        .mx_SetIntegrationManager_heading_manager {
+            display: flex;
+            align-items: center;
+            flex-wrap: wrap;
+            column-gap: $spacing-4;
 
-.mx_SetIntegrationManager > .mx_SettingsTab_heading > .mx_SettingsTab_subheading {
-    display: inline-block;
-    padding-left: 5px;
-    margin-top: 0px;
-}
+            .mx_SettingsTab_heading,
+            .mx_SettingsTab_subheading {
+                margin-top: 0;
+                margin-bottom: 0;
+            }
+        }
 
-.mx_SetIntegrationManager .mx_ToggleSwitch {
-    display: inline-block;
-    float: right;
-    top: 9px;
+        .mx_ToggleSwitch {
+            align-self: flex-start;
+            min-width: var(--ToggleSwitch-min-width); // avoid compression
```

Comment:
Why aren't we applying this globally?

Reply:
Since setting min-width to elements on a fixed width viewport globally might cause an unexpected regression, it would need analysis at first and I thought that task should be deferred to another PR.